user/test_main.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.testclient import TestClient 
+from .main import app 
+
+client= TestClient(app) 
+
+data = {"name": "user5","email":"user5@gmail.com","password":"user5"}
+
+#Testcase for post api
+def test_create():
+    response=client.post("/create",json=data)
+    if response.status_code == 400:
```

Comment:
We should test for all the possible responses by sending a failure input and check the response. Then on sending proper request input it should give 200.